src/components/sequencing/SequenceForm.svelte

```diff
@@ -285,7 +310,7 @@
         </select>
       </fieldset>
 
-      <fieldset>
+      <fieldset class="readonly">
```

Comment:
Does this class need to be dynamically added / removed?

Reply:
oops...no I just put it on the wrong fieldset and forgot to remove it